openpype/pipeline/farm/patterning.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+import re
+
+
+def match_aov_pattern(host_name, aov_patterns, render_file_name):
+    """Matching against a `AOV` pattern in the render files.
+
+    In order to match the AOV name we must compare
+    against the render filename string that we are
+    grabbing the render filename string  from the collection
+    that we have grabbed from `exp_files`.
+
+    Args:
+        app (str): Host name.
+        aov_patterns (list): List of AOV patterns from AOV filters.
+        render_file_name (str): Incoming file name to match against.
+
+    Returns:
+        bool: Review state for rendered file (render_file_name).
+    """
+    aov_pattern = aov_patterns.get(host_name, [])
+    if not aov_pattern:
+        return False
+    return re.match(aov_pattern, render_file_name) is not None
```

Comment:
Its crashing for me. I think it is because `re.match` accepts only string as a pattern, not list.

Now - `aov_patterns` are set by default like this:
```python
{'maya': ['.*([Bb]eauty).*'], 'nuke': ['.*'], 'aftereffects': ['.*'], 'celaction': ['.*'], 'harmony': ['.*']}
```
so in case of maya and my test, the return line becomes:

```python
return re.match(['.*([Bb]eauty).*'], "Second_N.1001.exr") is not None
```
and this will crash with:

```
// Error: pyblish.plugin : Traceback (most recent call last):
  File "E:\projects\openpype\3x\.venv\lib\site-packages\pyblish\plugin.py", line 522, in __explicit_process
    runner(*args)
  File "E:\projects\openpype\3x\openpype\modules\deadline\plugins\publish\submit_publish_job.py", line 850, in process
  File "E:\projects\openpype\3x\openpype\modules\deadline\plugins\publish\submit_publish_job.py", line 460, in _create_instances_for_aov
  File "E:\projects\openpype\3x\openpype\pipeline\farm\patterning.py", line 24, in match_aov_pattern
    return re.match(aov_pattern, render_file_name) is not None
  File "D:\gfx\Autodesk\Maya2022\Python37\lib\re.py", line 173, in match
    return _compile(pattern, flags).match(string)
  File "D:\gfx\Autodesk\Maya2022\Python37\lib\re.py", line 276, in _compile
    return _cache[type(pattern), pattern, flags]
TypeError: unhashable type: 'list'
```
Untested suggestion:
```suggestion
    return any(re.match(p, render_file_name) for p in aov_pattern)
```